dora/shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -148,7 +148,7 @@ public final class LoadCommand extends AbstractFileSystemCommand {
   public LoadCommand(FileSystemContext fsContext) {
     super(fsContext);
     AlluxioProperties properties = fsContext.getClusterConf().copyProperties();
-    properties.set(PropertyKey.DORA_CLIENT_READ_LOCATION_POLICY_ENABLED, false);
+    properties.set(PropertyKey.DORA_ENABLED, false);
```

Comment:
so we should somehow explain this command is deprecated and does not work anymore on the new architecture

Reply:
Change the code here to print out error messages to say this command is deprecated?

Reply:
will do